docs/user.md

```diff
@@ -454,3 +454,41 @@ monitoring is outside the scope of operator responsibilities. See
 [configuration reference](reference/cluster_manifest.md) and
 [administrator documentation](administrator.md) for details on how backups are
 executed.
+
+## TLS configuration
```

Comment:
```suggestion
## Custom TLS certificates
```

Configuration sounds like, it can be globally configured in the ConfigMap / OperatorConfiguration which isn't possible at the moment.